src/components/crc.js

```diff
@@ -12,18 +12,31 @@ class CRC extends React.Component {
     constructor (props) {
         super(props);
 
+         var regex = new RegExp("[\\?&]share=([^&#]*)"),
+        results = regex.exec(location.search);
+
+        // An array of cards from the url
+        let share = results == null ? "" : JSON.parse(atob(decodeURIComponent(results[1].replace(/\+/g, " "))));
```

Comment:
Just do `results ? JSON.parse(...) : ''`. Also, use single quotes instead of double quotes.